### ISSUE TITLE
Add Documentation

### ISSUE THREAD

**Issue by morganog** [morganog/CookNook#2]
- [ ] Make Code Easier to Understand with Comments
- [ ] Update ReadMe File to Contain More Information

**Comment by bhorsujal** [comment]
I would like to work on this issue in the upcoming hacktoberfest event.
Can you please assign the issue to me?

**Comment by morganog** [comment]
Yes! You can go ahead and work on this and I will assign it to you.

**Comment by abshek7** [comment]
I would like to work on this issue in the upcoming hacktoberfest event.
Can you please assign the issue to me?

**Comment by Eeshu-Yadav** [comment]
@morganog i want to make a pull request for this , can you assign it to me?

**Comment by aakashmohole** [comment]
can i work on this issue
